library/src/main/java/io/nlopez/clusterer/Clusterer.java

```diff
@@ -107,6 +107,16 @@ public void run() {
         }
     };
 
+    private Listener listener;
+
+    public Listener getListener() {
```

Comment:
The getter should be renamed, as getListener is too generic. See the comments about the interface name.